### PR TITLE
fix(ISIS): LS scales diamond stays in scales and deviation is now the same as the PFD

### LINKED ISSUE #8040: isis : LS indications go outside the scale if deviation greater than 2 dots
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2023-05-17T19:05:00+00:00",
    "ref": "refs/heads/experimental",
    "sha": "e2da961b511bf8d5c29524ff135cfbe41aa2d83f",
    "actor": "beheh",
    "event_name": "manual",
    "pretty_release_name": "experimental:e2da961b",
    "version": "v0.10.0-exp.e2da961"
}
```

### Describe the bug

As visible, I was two dots higher than the received G/S signal and as the deviation started to decrease, the glide slope diamond appeared outside the scale. Might be possible that the same happens laterally but didnt keep an eye for it 😅 
ISIS

### Expected behavior

LS indication to remain inside the scale

### Steps to reproduce

1) Press the LS key on the ISIS
2) Be more than two dots of deviation of the glideslope, for example, more than two above like the picture above and observe the diamond behaviour as the deviation decreases.

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

Bruno_pt99#5802

### PR DESCRIPTION
**Pull request by Maximilian-Reuter**
Fixes #8040

## Summary of Changes

The scaling of the deviation indicator was way off and has now been made the same as on the PFD scales.
~~Additionally the magenta diamond is now constrained inside the scales even when the real deviation is way off.
This is according to V1 simulations the correct indication.~~

EDIT: According to new references the Diamond is actually only half visible when out of scale. 

## Screenshots (if necessary)

**PREVIOUS**
PFD and ISIS LS mismatch

**NOW**
IMG_4446

## References

V1 Simulations - A320 Capt

IMG_4446

## Additional context

Discord username (if different from GitHub): Chaoz#3065

## Testing instructions

1. Get the aircraft ideally on a downwind to an ILS approach
2. Make sure that the ILS is tuned
3. Observe both the PFD and ISIS LS scales
4. When out of scale the ISIS magenta diamond should only show half at the edge of the scale just touching the tip against it
5. When inside scale both ISIS and PFD diamond position should be the same

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx [hunk 1]
```diff
@@ -7,23 +7,28 @@ type DeviationIndicatorProps = {
...
-        <line x1={100} x2={100} y1={2} y2={18} strokeWidth={4} stroke="white" />
-        <circle className="StrokeWhite NoFill" cx={145} cy={10} r={7} />
-        <circle className="StrokeWhite NoFill" cx={190} cy={10} r={7} />
-        { available
-            && (
-                <g transform={`translate(${(deviation * 50).toFixed(5)} 0)`}>
+const DeviationIndicator: React.FC<DeviationIndicatorProps> = ({ deviation, available }) => {
+    const maxDeviation = 1.778; // the maximum position the diamond can be while still fully inside scale
+
+    const dots = Math.sign(deviation) * Math.min(Math.abs(deviation / 0.4), maxDeviation);
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx:13]: This is not dots if the max deviation is less than 2?

Comment (Maximilian-Reuter) [reply fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx:13]: not sure I understand. Max deviation is in dots. 1.778 dots to be exact.

I do agree that the variable isnt perfectly named so I am open for suggestions.

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx:13]: The scale is +/- 2 dots. It doesn’t make it to the second dot?

**Comment by frenchairbuspilot** [comment]
Discord          : jp320#3336
Object of testing: #8044
Tier of Testing  : 1
Date : 28/05/2023

Testing Process: 
boot msfs rwy 25R EBBR, inserted a right hand downwind, via kerky ILS Z 25R no star and no SID out of 25R. tried a interception from above, go around and normal ILS. 


Negatives:
No route discontinuity after inserting the appr, should have one bewteen the runway extension wpt ( 1600ft here) the flight plan directly linked this waypointn to kerky ( should not be the case) , and after take off passing 1600ft I've inserted a DIR TO BUN ( waypoint after KERKY) and something went wrong after it displayed correctly on the ND, resulting in a loss of flight plan in the fmgc. weird.

For the ILS on ISIS, it seems that the diamond do not to have the same reoslution, meaning it moves faster than the one on the PFD ( glide only) for testing purpose i dove below the g/s isis was following immediatly i  used active pause  once below the g/s and saw that the glide on pfd was catching up during the pause to match the same position of the glide as the one on ISIS ( so same indication overall the PFD one is just lagging behind, also ISI glide comes alive way before PFD glide does. No issues on LOC.

Testing Results: 
Passed, delay on the PFD glide isn't that obvious in normal ops.

Conclusions: 
it would be perfect if the position of both glide indication matched but the slight delay isn't harmful.

**Comment by Maximilian-Reuter** [comment]
Still needs some work as new visual references were collected by V1
IMG_4446

**Review by tracernz — APPROVED** [review]

**Comment by BlueberryKing** [comment]
Dev Team Quality Assurance Report

Discord : BlueberryKing#3336
Object of testing: https://github.com/flybywiresim/a32nx/pull/8044
Tier of Testing : 1
Date : 04/06/2023

Testing Process:

- Loaded up at KOAK, took off, loaded in ILS 30.

- Flew onto the ILS, comparing indications on the PFD and the ISIS.

Negatives:

Testing Results:
Passed

Conclusions:
LGTM!

**Comment by tracernz** [comment]
Dev Team Quality Assurance Report

Discord : tracernz
Object of testing: https://github.com/flybywiresim/a32nx/pull/8044
Tier of Testing : 1
Date : 09/06/2023

Testing Process:
- Flew NZTG/NZAA on ILS23L.
  - Checked deviations either side from extremes right through on glideslope and LOC.

Negatives:
 - Nil

Testing Results:
Passed

Conclusions:
LGTM

**Pull request merged**